packages/client/tests/e2e/_utils/run.ts

```diff
@@ -88,8 +88,8 @@ async function main() {
     const dependencies = pkgJsonWithRuntimeDeps.dependencies as Record<string, string>
 
     // replace references to unbundled local packages with built and packaged tarballs
-    if (dependencies['@prisma/engines']) dependencies['@prisma/engines'] = '/test/prisma-engines-0.0.0.tgz'
-    if (dependencies['@prisma/debug']) dependencies['@prisma/debug'] = '/test/prisma-debug-0.0.0.tgz'
+    if (dependencies['@prisma/engines']) dependencies['@prisma/engines'] = '/tmp/prisma-engines-0.0.0.tgz'
+    if (dependencies['@prisma/debug']) dependencies['@prisma/debug'] = '/tmp/prisma-debug-0.0.0.tgz'
```

Comment:
It looks like the tests are passing because e2e tests are not tested on Windows.
How interesting it is to run these tests on Windows? I guess we could open a tech-debt issue to investigate.

Reply:
It doesn't unfortunately.

FWIW, it wouldn't be very useful to run these tests in Docker with the same base image, as that would be just Linux in a VM on Windows and not actually Windows. Windows containers do exist though (https://hub.docker.com/_/microsoft-windows-base-os-images), and if I understand correctly, don't require spawning a VM as long as the version of the container OS matches the host OS version. Maybe there's a way to use them without Docker (which will refuse to install without Hyper-V support)?

Reply:
We can make this run this without containers too, of course, especially if we want to test this across operating systems. Right now these tests assume Linux (even Debian-based Linux specifically), and Docker allows to run them on macOS or Windows or RHEL or Alpine but not actually test native support for those platforms.

Reply:
Containers are still useful for isolation, or if we want to be able to run tests for different targets with different base images on the same system, but not fundamentally required.